### PR TITLE
LCU Update October 2022

### PR DESCRIPTION
**Pull request by mtelvers**
Microsoft Updates

### CODE REVIEW

File: src/conf.ml [hunk 1]
```diff
@@ -73,7 +73,7 @@ let distros = Distro.(active_distros `X86_64 |> List.filter (fun d ->
 
 let arches_for ~distro = Distro.distro_arches Ocaml_version.Releases.latest distro
 
-let win10_revision : Distro.win10_lcu = `LCU20220913
+let win10_revision : Distro.win10_lcu = `LCU20221011
```
Comment (MisterDA) [inline src/conf.ml:76]: Why don't we use `Dockerfile_opam.Distro.win10_current_lcu` here?

**Review by MisterDA — APPROVED** [review]

File: src/conf.ml (hunk 1 above)
Comment (MisterDA) [reply src/conf.ml:76]: (maybe I've already asked? can't recall)

Comment (mtelvers) [reply src/conf.ml:76]: Looks like a good idea to me.  One less change to make each month.  @dra27 any reason not to do this?

Comment (MisterDA) [reply src/conf.ml:76]: For reference it was added as part of https://github.com/ocurrent/docker-base-images/pull/124 which is all David's work but he didn't point to `win10_current_lcu`. I think it's ok to do so.

**Pull request merged**